src/nav/MainNavBar.js

```diff
@@ -18,6 +18,12 @@ import {
   getTitleTextColor,
 } from '../selectors';
 
+const componentStyles = StyleSheet.create({
+  hidden: {
```

Comment:
Left it hidden because presence or absence of this component on the extreme right changes how the MainNavBar is laid out. Figuring out a way to render the MainNavBar properly even when People Nav Button is not rendered 👍